### PR TITLE
Only check unsupported browsers on user rendering

### LINKED ISSUE #10880: Login broken on server master
See https://github.com/nextcloud/server/pull/26358#issuecomment-1287009773

cc @tobiasKaminsky

### PR DESCRIPTION
**Pull request by skjnldsv**
Fix https://github.com/nextcloud/server/pull/26358#issuecomment-1287699542

### CODE REVIEW

File: core/Application.php [hunk 1]
```diff
@@ -310,6 +312,7 @@
 			}
 		);
 
+		$eventDispatcher->addServiceListener(BeforeTemplateRenderedEvent::class, BeforeTemplateRenderedListener::class);
```
[the remark this answers is not in the document]
Comment (skjnldsv) [reply core/Application.php:316]: Seems irrelevant? :thinking:

**Review by AlvaroBrey — APPROVED** [review]
Can confirm the mobile app can log in properly with this branch

**Review by szaimen — APPROVED** [review]
I fear I can only give a 🐘.

Not sure if that is enough...

**Comment by szaimen** [comment]
Also conflicts

**Review by tobiasKaminsky — APPROVED** [review]

**Pull request merged**